### PR TITLE
improved env example

### PR DESCRIPTION
**Pull request by kidig**
Improved env.example and readme for better user experience.

### CODE REVIEW

File: node/src/truv.js [hunk 1]
```diff
@@ -2,6 +2,13 @@ import fetch from "node-fetch";
 
 const { API_CLIENT_ID, API_SECRET, API_PRODUCT_TYPE } = process.env;
 
+if (!API_CLIENT_ID || !API_SECRET) {
```
Comment (busyerakli) [inline node/src/truv.js:5]: Also add check for API_PRODUCT_TYPE here

**Review by busyerakli — APPROVED** [review]

File: node/src/truv.js (hunk 1 above)
Comment (busyerakli) [reply node/src/truv.js:5]: And don't forget to copy this change to other languages

Comment (kidig) [reply node/src/truv.js:5]: > Also add check for API_PRODUCT_TYPE here

it's already checked here: https://github.com/truvhq/quickstart/blob/master/node/src/serve.js

Comment (kidig) [reply node/src/truv.js:5]: > And don't forget to copy this change to other languages

I checked them all, it seems ok, but except c-sharp...
Unfortunately, this lang is not my cup of tea. :(

File: .env.example [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
-#API_PRODUCT_TYPE=admin
\ No newline at end of file
+# please set your <Client ID>
+API_CLIENT_ID=
+
+# please set your <Access key>
+API_SECRET=
+
+# you can choose a different product type, just uncomment the line
+API_PRODUCT_TYPE=employment
```
Comment (chetvil) [inline .env.example:8]: could we add `dds` and `pll` as well?

**Review by chetvil — APPROVED** [review]

File: .env.example (hunk 2 above)
Comment (kidig) [reply .env.example:8]: ok. I'll check that its available for each languages

Comment (kidig) [reply .env.example:8]: @chetvil done ;)

**Pull request merged**